ElrondNetwork/api.elrond.com
# Operations for sc execution

## Type

## Problem setting
- There is no possibility to show smart contract execution log
  
## Proposed Changes
- Decode writeLog and signalError logs from smart contract execution

## How to test
- Check if this transactions 538454458bb8f6fbc23cadd891a29e1c368faebf427a7d30ee2f16a62d75689d , 1c5b7dc9c4c1653abe718be9b04553b5e5b215c3da362f3527f1f8e6512dc9da on testnet and check if last operation is a signalError or writeLog operation
